src/components/ResetPassword/index.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react';
+import ImageFader from '../Common/ImageFader';
+import { defineMessages, useIntl } from 'react-intl';
+import LanguagePicker from '../Layout/LanguagePicker';
+import Button from '../Common/Button';
+import { Field, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import axios from 'axios';
+import { useRouter } from 'next/router';
+
+const messages = defineMessages({
+  resetpassword: 'Reset your password',
+  password: 'Password',
+  confirmpassword: 'Confirm Password',
+  validationpasswordrequired: 'Password required',
+  validationconfirmpasswordrequired: 'Confirm Password required',
+  validationpasswordmatch: 'Password must match',
+  gobacklogin: 'Go Back to Login Page',
+  successmessage:
+    'If the link is valid and is connected to a user then the password has been reset.',
```

Comment:
Should we even allow the form to be displayed if the link isn't valid?

Reply:
Yes, otherwise you could brutoforce correct links by visiting the url. That's why it always gives the success message and tells you it will only really be successful if what you're doing is correct.

Reply:
Ah okay, that makes sense! Could we change the message slightly, to "If the password reset link was valid, the password has been successfully reset."  (This will be my last comment, you're awesome and thanks for being so great about the feedback 😀)